twin/incoming-voice.php

```diff
@@ -43,6 +43,8 @@
 	);
 }
 
+$gather->say("or press 0 for voicemail");
```

Comment:
Avoid hard coding text - add to config.php
Use the same voice as the other prompts